go.mod

```diff
@@ -1,5 +1,9 @@
 module github.com/thought-machine/please
 
+replace (
+    github.com/bazelbuild/remote-apis-sdks => github.com/peterebden/remote-apis-sdks master
```

Comment:
eeeeeh we sure we want to do this? there is some... experimentashone happening in there

Reply:
Eeeep sorry I didn't realise this ended up here! I was just trying to get `go install` to work! This was only meant to exist on tatskaari/master